### PR TITLE
Bump march hare to 2.20.0

### LINKED ISSUE #76: No error if RabbitMQ goes down
Before version 3.3.0, when Rabbit server went down, logstash throws:

```
Queue subscription ended! Will retry in 5s
RabbitMQ connection down, will wait to retry subscription
```

With version 4.0.1, nothing happen (as you can on docker-compose logs, here https://gist.github.com/ebuildy/d405dab7dbaca2e3a03dc2a7fa6f5b29) if queue server is down, is it normal?

Thanks you,

### PR DESCRIPTION
**Pull request by andrewvc**
This fixes https://github.com/logstash-plugins/logstash-input-rabbitmq/issues/76

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,4 +1,6 @@
 * 4.2.0
```
Comment (jsvd) [inline CHANGELOG]: this should be 4.2.1, no?

**Review by jsvd — CHANGES_REQUESTED** [review]

File: CHANGELOG (hunk 1 above)
Comment (andrewvc) [reply CHANGELOG]: Yes!

Comment (andrewvc) [reply CHANGELOG]: I think you left this comment on the wrong line BTW.

**Review by jsvd — APPROVED** [review]

**Comment by jsvd** [comment]
LGTM

**Comment by digitalkaoz** [comment]
@andrewvc  cant you tag a release?

**Pull request closed without merging**